### DISCUSSION TITLE
Jewelry (rings & necklaces) can be worn without any limits

### DISCUSSION THREAD

**Discussion by vorpal-void** [cataclysmbn/Cataclysm-BN#3238] [General]
### Describe the bug

Any character can wear not two, not ten, but ALL of the necklaces and rings at once.
In the least innocuous form it doesn't break anything, except that it allows you to have a stable morale boost due to stylish clothing, but there's a +8 point limit there as it is.
Worse case scenario, when such items are added by mods and create gameplay changes, like Magiclysm - then it creates an opportunity to break everything very fast.

### Steps To Reproduce

1. Start new world
2. Debug spawn all of the rings and amulets
3. Wear them all, see no problem? Lookin' fab
4. Start new world, with Magiclysm, spawn +N rings or find them naturally if you have time to actually play the game
5. Wear all of them, become demigod

### Expected behavior

This behavior should be limited, but remain logical. 

DDA implemented this balance [via new flag](https://github.com/CleverRaven/Cataclysm-DDA/pull/47518) that allows user to wear only one item per layer. This solution works and may be useful to port for other stuff, but doesn't seem like complete fix because from infinite rings we're coming to only two, like it's Skyrim [or something](https://i.redd.it/edyfem4c7m191.png). I don't think we have some way to count player fingers or see if they have fingers at all and not some tentacles or other limb-altering mutations, but it would be better to actually count the possibilities for additional wearables than artificially limiting them without any logical sense.

### Screenshots

![jewelry](https://user-images.githubusercontent.com/60776130/170548126-fbe9e34b-86fd-4f42-80d6-eea97cc39c6c.gif)

### Versions and configuration

- OS: Windows
    - OS Version: 10.0 2009
- Game Version: 9625ba1 [32-bit]
- Graphics Version: Tiles
- Game Language: English [en_US]
- Mods loaded: [
    Bright Nights [dda]
]

### Additional context

At the very least, jewelry could use some encumbrance since all of the demo above virtually does not exists and does not weight anything while being equipped: 

![изображение](https://user-images.githubusercontent.com/60776130/170550222-ccf72b26-531b-403f-ae09-a2864c4da36c.png)

**Comment by Proxiehunter** [comment]
It feels like another possible option would be for the creators of mods like Magiclysm and Arcana to code it so you can only have so many _effects_ from jewelry at once so your character can still bling themselves out like Mr. T but not break the game. Not sure how difficult the coding would be to do that though.

**Comment by RodriguesCIA** [comment]
The effects of unlimited jewelry is already limited mechanically -- for Stylish characters. You can get only so much out of it before you have to start wearing fancy clothing.

And some jewelry do cause encumbrance: dental grills. That said, only some of them, and it seems to calculate its morale effect for Stylish differently from regular jewelry.

**Comment by Proxiehunter** [comment]
> The effects of unlimited jewelry is already limited mechanically -- for Stylish characters. You can get only so much out of it before you have to start wearing fancy clothing.

Their main worry doesn't seem to be stylish characters but effects from mods like Magiclysm and Arcana where you can wear jewelry that gives you stat bonuses or has other effects. And it's an understandable one. Although I think Magiclysm has already implemented a limit on the number of enchanted rings you can wear, but that might not be in the Bright Nights version of the mod.

**Comment by leoCottret** [comment]
The rings of magiclysm already have a limit, they have the "only_one" tag, so you can't wear 2 +4 STR rings for example.
One could wear 10 rings and maybe 10 bracelets too, setting the limit to 1 per layer seems overkill.
Changing it to balance since it's not really a bug but a balance change suggestion.
